src/computer.h

```diff
@@ -116,6 +116,8 @@ class computer
     private:
         // Difficulty of simply logging in
         int security;
+        // Date of next attempt
+        long next_attempt = 0;
```

Comment:
> I'm a bit worry about size of  int  (I'm not sure if its size is enough for long-run game), 

Yes, but `calender` only exports an `int` anyway. If there's an overflow, it already happens when using the calender (e.g. when incrementing the turn number). In other words: the long here will *always* contain a value that fits in an `int`: `int(calendar::turn) + 450` is an `int` and gets stored in this `long`.

In fact, when overflow happens, it happens *before* the value is stored and overflow of signed int is undefined behavior anyway.